Sources/Identity/SubscriptionInfo.swift

```diff
@@ -78,6 +78,9 @@ import Foundation
     /// Management purchase URL
     @_spi(Internal) public let managementURL: URL?
 
+    /// The display name of the subscription as configured in the RevenueCat dashboard.
+    @objc public let displayName: String?
```

Comment:
Just to confirm: do we need to make this property public or is it only supposed to be used by RevenueCatUI (e.g. `@_spi`)?

Reply:
I'll revert this for now. It's definitely out of scope of the PR and deserves a broader conversation 👍